src/pages/blog/[slug].astro

```diff
@@ -117,15 +118,27 @@ const structuredData = {
 				</div>
 			</header>
 			<hr class="-mx-16 my-8 border-astro-gray-500 lg:my-16" />
-			<div class="2xl:flex">
+			<div class="flex flex-wrap gap-y-12 lg:gap-y-16 max-w-prose">
 				<article class="prose">
 					<Content />
 				</article>
-				<div class="border-t border-astro-gray-500 mt-12 2xl:w-0 flex flex-col 2xl:before:h-full">
-					<div class="my-12 2xl:ml-[9rem] 2xl:w-56 2xl:sticky 2xl:bottom-12">
+				<hr class="2xl:hidden -mx-16 basis-[calc(100%+8rem)] border-astro-gray-500" />
+				<div
+					class="w-full 2xl:w-0 2xl:mt-24 flex flex-col 2xl:before:h-full"
+				>
+					<div class="2xl:ml-[9rem] 2xl:w-56 2xl:sticky 2xl:bottom-12">
 						<Promo />
 					</div>
 				</div>
+				{
+					data.related.length > 0 && (
+						<hr class="-mx-16 basis-[calc(100%+8rem)] border-astro-gray-500" />
+						<nav aria-labelledby="_related-posts">
+							<h2 class="heading-3" id="_related-posts">Previously…</h2>
```

Comment:
I do wonder whether "Related Posts" makes more sense as a heading, given that these won't *always* be releases in a sequential order? e.g. If we have a more community type of post, I'm not sure "Previously... how Astro does i18n" *exactly* fits vs just saying, "Here's something sort of random but in that ballpark" -- like the Community Loaders and the specific loader blog posts that weren't *actually* previous, but released at the same time?

```suggestion
							<h2 class="heading-3" id="_related-posts">Related Posts</h2>
```
Don't feel super strongly, just pointing it out!

Reply:
Perfect, thank you!